pkgs/development/libraries/libxslt/default.nix

```diff
@@ -42,22 +44,22 @@ stdenv.mkDerivation rec {
     libxml2.py
     python
     ncurses
-  ] ++ lib.optionals cryptoSupport [
-    libgcrypt
   ];
 
   propagatedBuildInputs = [
     findXMLCatalogs
   ];
 
+  LIBGCRYPT_CFLAGS = lib.optionalString cryptoSupport "-I${libgcrypt.dev}/include";
+  LIBGCRYPT_LIBS = lib.optionalString cryptoSupport "-L${libgcrypt}/lib -lgcrypt";
```

Comment:
Those envs are overwriten by the autoconf script which then fails to link when cross compiling

nix build .#pkgsCross.aarch64-multiplatform.libxslt
```
libxslt>   CC       date.lo
libxslt>   CC       saxon.lo
libxslt>   CC       dynamic.lo
libxslt>   CCLD     libexslt.la
libxslt> /nix/store/a0cl8qcfhsidgdxx371inwlg0m885zys-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: /nix/store/yqrjnp7b4s2746c3h7jh5zgac3iv2igg-libgcrypt-1.10.2/lib/libgcrypt.so: error adding symbols: file in wrong format
libxslt> collect2: error: ld returned 1 exit status
libxslt> make[2]: *** [Makefile:508: libexslt.la] Error 1
libxslt> make[2]: Leaving directory '/build/libxslt-1.1.37/libexslt'
libxslt> make[1]: *** [Makefile:669: all-recursive] Error 1
libxslt> make[1]: Leaving directory '/build/libxslt-1.1.37'
libxslt> make: *** [Makefile:451: all] Error 2
```